packages/@n8n/config/src/configs/logging.config.ts

```diff
@@ -1,6 +1,10 @@
 import { Config, Env, Nested } from '../decorators';
 import { StringArray } from '../utils';
 
+export const LOG_SCOPES = ['executions', 'license', 'scaling'] as const;
```

Comment:
Would be nice to have proper description what do these scopes cover so we know when to e.g. use that scope when developing something new

Reply:
[b71e961](https://github.com/n8n-io/n8n/pull/11127/commits/b71e9615c80396cbb600067502bc3fe5cd37eced)